app/src/main/java/ingsw/group1/findmyphone/cryptography/RandomStringGenerator.java

```diff
@@ -0,0 +1,53 @@
+package ingsw.group1.findmyphone.cryptography;
+
+import java.security.SecureRandom;
+import java.util.Random;
+
+/**
+ * Class used to generate random strings and int value within the bounds.
+ *
+ * @author Pardeep Kumar
+ */
+ class randomStringGenerator {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'randomStringGenerator' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/RicDeZen/ingsw-group1-findmyphone/pullRequest?prid=4958670)

Reply:
What i've to change here to remove this issue?

Reply:
When Codacy finds a class containing only `static` methods or constants, it suggests that it should be called [Something]Utils or [Something]Helper.
So either rename it to that, or make it a normal object class that can be instantiated (see RandomSMSPeerGenerator or just java's Random)

Reply:
Now i understand, thanks